saleor/graphql/core/mutations.py

```diff
@@ -259,7 +259,7 @@ def _save_m2m(cls, info, instance, cleaned_data):
                 f.save_form_data(instance, cleaned_data[f.name])
 
     @classmethod
-    def user_is_allowed(cls, user, input):
+    def user_is_allowed(cls, user):
```

Comment:
I was almost sure that we used `input` argument in one of our mutations to determine user's permission based on the input data. But that might have changed in the meantime so if we are sure that it's unused, we can drop it.

Reply:
Yes, definitively sure about that.